.zuul.yaml

```diff
@@ -48,6 +48,17 @@
     pre-run: playbooks/setup-env-restricted.yaml
     run: playbooks/unit-test.yaml
 
+- job:
+    name: system-test-arch-commands-options
+    description: Run Toolbx's commands-options system tests on Arch Linux
+    timeout: 6300
+    nodeset:
+      nodes:
+        - name: arch
+          label: arch-linux
+    pre-run: playbooks/arch/setup.yaml
```

Comment:
so if something will fail here, you might have RETRY_LIMIT.
Maybe try to do:
```
run:
  - playbooks/arch/setup.yaml
  - playbooks/system-test-commands-options.yaml
```

Reply:
It was intentional to use `pre-run` for downloads that can fail due to passing network problems, outdated caches, etc..

We use `zuul.attempts > 1` to decide if the DNF cache needs to be forcibly updated on Fedora Rawhide nodes, or to resolve problems arising from the DNF5 transition.

Downloading the Go modules is another thing that has proven to be flaky and benefits from this.  It's still not 100% robust, but a lot more than what it used to be.